idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/coercions/explicit/locaExplicitCoercionOnKtProperty.kt

```diff
@@ -17,18 +18,20 @@ import org.jetbrains.kotlin.types.KotlinType
 val IdeMetaPlugin.localExplicitCoercionOnKtProperty: ExtensionPhase
   get() = addLocalInspection(
     inspection = explicitCoercionKtProperty,
-    level = HighlightDisplayLevel.WEAK_WARNING,
-    groupPath = ProofPath + arrayOf("Coercion")
+    groupPath = ProofPath + arrayOf(MetaIdeBundle.message("proofs.coercion")),
+    groupDisplayName = MetaIdeBundle.message("proofs.coercion"),
+    level = HighlightDisplayLevel.WEAK_WARNING
   )
 
-const val COERCION_EXPLICIT_PROP = "Coercion_explicit_prop"
+const val COERCION_EXPLICIT_PROP = "CoercionExplicitProp"
 
 val IdeMetaPlugin.explicitCoercionKtProperty: AbstractApplicabilityBasedInspection<KtProperty>
   get() = applicableInspection(
     defaultFixText = COERCION_EXPLICIT_PROP,
+    staticDescription = MetaIdeBundle.message("proofs.coercions.inspection.explicit.property.static.description"),
     inspectionHighlightType = { ProblemHighlightType.INFORMATION },
     kClass = KtProperty::class.java,
-    inspectionText = { "Not used at the moment because the highlight type used is ProblemHighlightType.INFORMATION" },
+    inspectionText = { MetaIdeBundle.message("proofs.coercions.inspection.explicit.property.inspection.text") },
```

Comment:
I'd personally revert these, since this one is a disclaimer for the developer given the text will not show up at runtime because `ProblemHighlightType.INFORMATION`, not a string meant for end users to see. Developers likely know English pretty broadly so I'd just keep these like they were, so we keep the disclaimer as explicit as possible and as close as possible to where you're writing the code so you don't miss it when working over these feature.